src/types/QueryParams.ts

```diff
@@ -4,5 +4,6 @@ export type QueryParams = {
   orderBy?: string
```

Comment:
I would assume this file is only for the cases list page. Should we prefix the filename with the page name to be more specific?

Reply:
Refactored in [b22d455](https://github.com/ministryofjustice/bichard7-next-ui/pull/24/commits/b22d455b17a38c3e89b99d3442fa71a1e35c2587)